camel/societies/role_playing.py

```diff
@@ -287,12 +287,16 @@ def init_agents(
                 agents.
         """
         if self.assistant_functions is not None:
-            assistant_config = FunctionCallingConfig.from_openai_function_list(
-                function_list=self.assistant_functions,
-                function_call="auto",
-            )
+            assistant_model_config = (
+                FunctionCallingConfig.from_openai_function_list(
+                    function_list=self.assistant_functions,
+                    function_call="auto",
+                ))
+        elif assistant_agent_kwargs:
+            assistant_model_config = assistant_agent_kwargs.pop(
+                "model_config", None)
         else:
-            assistant_config = None
+            assistant_model_config = None
```

Comment:
We might want to remove all these related to `configs` and `assistant_functions` https://github.com/camel-ai/camel/pull/249 since they can be passed from `*_agent_kwargs`.